docs/networking/best-practice.md

```diff
@@ -0,0 +1,198 @@
+
+2. Network connection between VM networks and the management network
+
+  A feasible method to ensure network connectivity between VM networks and the management network is to manually add static routes. The following commands add static routes on the router to allow VMs in the untagged network, VLAN 100 and VLAN 200 to access the management network.
+
+  ```
+  router(config)# config terminal
+  router(config)# ip route 10.10.0.0 255.255.255.0 ethernet0/1
+  router(config)# ip route 192.168.0.0 255.255.255.0 ethernet0/2
+  router(config)# ip route 192.168.100.0 255.255.255.0 ethernet0/2
+  router(config)# ip route 192.168.200.0 255.255.255.0 ethernet0/2
+  router(config)# end
+  ```
+
+  The route table would be like this:
+
+  ```
+  Router#show ip route
+  Codes: C - connected, S - static, R - RIP, M - mobile, B - BGP
+         D - EIGRP, EX - EIGRP external, O - OSPF, IA - OSPF inter area
+         N1 - OSPF NSSA external type 1, N2 - OSPF NSSA external type 2
+         E1 - OSPF external type 1, E2 - OSPF external type 2
+         i - IS-IS, su - IS-IS summary, L1 - IS-IS level-1, L2 - IS-IS level-2
+         ia - IS-IS inter area, * - candidate default, U - per-user static route
+         o - ODR, P - periodic downloaded static route
+
+  Gateway of last resort is not set
+
+  C    192.168.200.0/24 is directly connected, Ethernet0/2.200
+     10.0.0.0/24 is subnetted, 1 subnets
+  C       10.10.0.0 is directly connected, Ethernet0/1
+  C    192.168.0.0/24 is directly connected, Ethernet0/2
+  C    192.168.100.0/24 is directly connected, Ethernet0/2.100
+  ```
+
+## Access Harvester Load Balancers from Different VLANs
+The Harvester load balancer is divided into two types: VM load balancer and guest Kubernetes cluster load balancer. 
+
+1. The load balancer IP of the VM load balancer is only exposed within the same network as the Harvester hosts, or in other words, the management network. To access the VM load balancer from outside the network, you have to prove the route from outside client to the management network. For example, if the VM load balancer `lb1` has obtained its load balancer IP var DHCP and you want to access it from the VM `VM5`, you need to add the following static route on the router to
+
+  ```
+  router(config)# ip route 10.10.0.0 255.255.255.0 ethernet0/1
+  router(config)# ip route 192.168.0.0 255.255.255.0 ethernet0/2
+  ```
+
+2. The load balancer IP of the guest Kubernetes cluster load balancer is exposed within the VM network. In the diagram above, the guest cluster `demo` is within the VM network `vlan200` because the VMs consist of the guest cluster are in the `vlan200`. Thus, the guest Kubernetes cluster load balancer `lb2` is exposed within the VM network `vlan200`. There are three scenarios to explain how to access `lb2` if it has obtained the load balancer IP via DHCP:
```

Comment:
```suggestion
2. The load balancer IP of the guest Kubernetes cluster load balancer is exposed within the VM network. In the diagram above, the guest cluster `demo` is within the VM network `vlan200` because the VMs consisting of the guest cluster are in the `vlan200`. Thus, the guest Kubernetes cluster load balancer `lb2` is exposed within the VM network `vlan200`. There are three scenarios to explain how to access `lb2` if it has obtained the load balancer IP via DHCP:
```